test/parser/CMakeLists.txt

```diff
@@ -1,2 +1,2 @@
 addLinkAndDiscoverTest(ParallelBufferTest parser)
-addLinkAndDiscoverTest(LiteralOrIriTypeTest)
\ No newline at end of file
+addLinkAndDiscoverTest(LiteralOrIriTest)
```

Comment:
Add a newline to the end of this file (clang-format doesn't format the CMake files).